scripts/fast-reboot

```diff
@@ -245,7 +250,7 @@ function backup_database()
 {
     debug "Backing up database ..."
 
-    if [[ "$REBOOT_TYPE" = "fastfast-reboot" || "$REBOOT_TYPE" = "fast-reboot" ]]; then
+    if [[ "$REBOOT_TYPE" = "fastfast-reboot" || "$REBOOT_TYPE" = "express-reboot" || "$REBOOT_TYPE" = "fast-reboot" ]]; then
         # Advanced reboot: dump state to host disk
         sonic-db-cli ASIC_DB FLUSHDB > /dev/null
```

Comment:
Shouldn't the ASIC_DB be cleared during express-reboot? There might be 2 issues if it isn't cleared:

1. It might have stale entries after SWSS has replayed CONFIG/APPL DB entries.
2. HIDDEN keys would be present which will set m_veryFirstRun = False in Syncd.cpp. As a result, onApplyViewInFastFastBoot() won't be called during APPLY_VIEW notification processing.

Reply:
Yes asic_db is cleared during express-boot:
sonic-db-cli ASIC_DB FLUSHDB > /dev/null

Reply:
Thank you for the clarification.